accounting/static/vendor/css/bootstrap.min.css

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
Can you download external dependencies rather than committing them to the repo? You have snippets for this in Ocim that you can reuse.

Reply:
Yes. I had done this originally because for some reason I had to pass in the CSS to wkhtmltopdf directly which I think wasn't able to get the link tags to work. But I solved the issues causing this eventually, so we can remove this and use `link`.

Reply:
Actually, can we avoid doing this? I remembered that whenever `static` was called in Ocim dev for me it would take too long when my internet wasn't the fastest, to download everything. I don't see us changing the use of this Bootstrap version at all, since any greater version is going to break `wkhtmltopdf`.

Reply:
Downloading it from the source is the surest way to ensure that we are using the actual upstream version - when it's just committed locally it's much less clear what version is being used at times, and it has been modified.

I agree with your point about downloading them every time though, it is a pain when the list grows. Let's go with the way you did it then, and we can always change if we get bitten.

Reply:
Thanks! Yes I agree those are pain points for committing locally, but luckily in this case the version is on the first few lines with copyright and license (not always the case of course), and the source is minified so it's highly unlikely that someone changed it without notice.